tokio/src/net/tcp/listener.rs

```diff
@@ -216,11 +216,13 @@ impl TcpListener {
     ///
     /// # Panics
     ///
-    /// This function panics if thread-local runtime is not set.
+    /// This function panics if there is no current reactor set, if the `rt`
+    /// feature flag is not enabled, or if thread-local runtime is not set.
```

Comment:
Setting a current reactor or thread-local runtime is the same thing?

Reply:
OK, and is my understanding correct that this also covers needing `enable_io` on the runtime?

Does it make sense to keep the description of the case that the `rt` feature flag is not enabled? Or would it be better to leave the comment as it was (here and in all the other places I changed it)?

Reply:
Ah, I see what you meant. You can just say that it panics if you are not inside a runtime with IO enabled. You can't be inside such a runtime if `rt` is not enabled, so we don't need to mention that.

Reply:
I've tried to write something that makes sense. Please let me know if it needs changing.